qiskit/ignis/verification/randomized_benchmarking/rb_utils.py

```diff
@@ -46,9 +46,9 @@ def count_gates(qobj, basis, qubits):
     Additional Information:
         nQ gates are counted in each qubit's set of gates.
     """
-    warn('The function `count_gates` will be deprecated. '
-         'Gate count is integrated into `gates_per_clifford` function.',
-         DeprecationWarning)
+    warn('The function `count_gates` will be deprecated. \
```

Comment:
I noticed signature of `QiskitError` is `*message` and we can feed multiple lines, but that of [warnings](https://docs.python.org/3/library/warnings.html#warnings.warn) is `message, category=None, stacklevel=1, source=None`. for warnings we need to feed one long message with `\`.

Reply:
This isn't actually the case, @chriseclectic is correct the `\` here is not necessary and I'm actually surprised that one of the linters didn't complain about it. You can break the string over multiple lines as long as it's inside parenthesis. It will automatically concatenate any strings until the first `,` which indicates the next argument. This is pretty easy to test in the Python REPL too if you want to test it.

Reply:
@mtreinish Thanks for following up. I ran local tox -epy36 test and it caused an error due to this (as far as I remember that was a type error). I tried to reproduce this but the test passed after git clean -fdX. Maybe something odd was happening in my local env. If you want I'll make another PR to remove `/`.